backends/platform/sdl/posix/posix.h

```diff
@@ -28,7 +28,7 @@
 class OSystem_POSIX : public OSystem_SDL {
 public:
 	// Let the subclasses be able to change _baseConfigName in the constructor
-	OSystem_POSIX(Common::String baseConfigName = ".scummvmrc");
+	OSystem_POSIX(Common::String baseConfigName = "scummvm.ini");
```

Comment:
I want to get rid of this parameter anyway. Platforms having a different config file path should simply override `getDefaultConfigFileName`. SamsungTV really abused this by what looks like supplying an absolute path....